fe/package.json

```diff
@@ -2,15 +2,15 @@
   "name": "tm-entity-store-ui",
   "version": "0.1.0",
   "private": true,
+  "main": "dist/index.html",
```

Comment:
FYI: If `main` is missing or does not point to an existing file `BE/require('tm-entity-store-ui')` fails